frontend/views/containers/dashboard/GroupMembersActivity.vue

```diff
@@ -88,18 +87,43 @@
 </template>
 
 <script>
+import { mapGetters } from 'vuex'
 import Tooltip from '@components/Tooltip.vue'
 
 export default ({
   name: 'GroupMembersActivity',
+  components: {
+    Tooltip
+  },
   data () {
     return {
       isReady: false,
       history: []
     }
   },
-  components: {
-    Tooltip
+  computed: {
+    ...mapGetters([
+      'groupStreaks',
+      'globalProfile'
+    ]),
+    onTimeStreaks () {
+      const membersOnStreak = []
+      const getDisplayName = name => this.globalProfile(name).displayName || name
+
+      for (const username in this.groupStreaks.onTimePayments) {
+        if (this.groupStreaks.onTimePayments[username] >= 2) {
+          membersOnStreak.push(getDisplayName(username))
+        }
+      }
+
+      const len = membersOnStreak.length
+      return len === 0
+        ? null
+        : {
+            members: membersOnStreak,
+            tooltipText: len === 1 ? this.L(`${len} member`) : this.L(`${len} members`)
```

Comment:
@Silver-IT In case you didn't realise it, [here](https://github.com/okTurtles/group-income/blob/master/frontend/common/translations.js#L10) is the reference.

Reply:
Yeah, that makes sense.